packages/flutter/lib/src/widgets/scroll_delegate.dart

```diff
@@ -225,7 +225,7 @@ abstract class SliverChildDelegate {
 }
 
 class _SaltedValueKey extends ValueKey<Key> {
-  const _SaltedValueKey(super.key);
+  const _SaltedValueKey(super.value);
```

Comment:
I know this doesn't change how anything works, just the naming, but if you only saw this line, it looks like it would be a significant change. Seems like a good lint to have.